src/catalog/catalog_structs.cpp

```diff
@@ -37,6 +38,16 @@ string getRelMultiplicityAsString(RelMultiplicity relMultiplicity) {
     }
 }
 
+string TableSchema::getPropertyName(property_id_t propertyID) const {
+    for (auto& property : properties) {
```

Comment:
consider indexing properties in TableSchema

Reply:
As we discussed, let's not do it now.